ui/src/app/pages/homepage/data-access-requirements.tsx

```diff
@@ -225,28 +227,57 @@ export const allModules: AccessModule[] = [
   duccModule,
 ];
 
-const LoginGovTooltip = () => <TooltipTrigger
-    content={'For additional security, we require you to verify your identity by uploading a photo of your ID.'}>
-  <InfoIcon style={{margin: '0 0.3rem'}}/>
-</TooltipTrigger>;
-
-// TODO merge with RegistrationTasks after we remove RegistrationDashboard
-const moduleLabels: Map<AccessModule, JSX.Element> = new Map([
-  [AccessModule.TWOFACTORAUTH, <div>Turn on Google 2-Step Verification</div>],
-  [AccessModule.RASLINKLOGINGOV, <div>Verify your identity with Login.gov <LoginGovTooltip/></div>],
-  [AccessModule.ERACOMMONS, <div>Connect your eRA Commons account</div>],
-  [AccessModule.COMPLIANCETRAINING, <div>Complete <AoU/> research Registered Tier training</div>],
-  [AccessModule.DATAUSERCODEOFCONDUCT, <div>Sign Data User Code of Conduct</div>],
-]);
-
-const externalSyncActions: Map<AccessModule, Function> = new Map([
-  [AccessModule.TWOFACTORAUTH, async() => await profileApi().syncTwoFactorAuthStatus()],
-  [AccessModule.RASLINKLOGINGOV, () => redirectToRas(false)],
-  [AccessModule.ERACOMMONS, async() => await profileApi().syncEraCommonsStatus()],
-  [AccessModule.COMPLIANCETRAINING, async() => await profileApi().syncComplianceTrainingStatus()],
-  // DUCC state is strictly internal to the RW
-  [AccessModule.DATAUSERCODEOFCONDUCT, () => {}],
-]);
+interface AccessModuleConfig {
+  isEnabledInEnvironment: boolean;  // either true or dependent on a feature flag
+  darLabel: JSX.Element;
+  externalSyncAction?: Function;
+}
+
+// This needs to be a function, because we want it to evaluate at call time,
```

Comment:
this comment is mostly copied from RegistrationTasks.  I'm not sure how much of it is still relevant.